index.js

```diff
@@ -15,13 +15,14 @@
 "use strict";
 
 const core = require("apollo-server-core");
+const { GraphQLUpload } = require("graphql-upload");
 const { ApolloServer } = require("./src/ApolloServer");
 const ApolloService = require("./src/service");
 const gql = require("./src/gql");
 
 module.exports = {
 	// Core
-	GraphQLUpload: core.GraphQLUpload,
+	GraphQLUpload,
```

Comment:
To avoid any potential for confusion, please move this export to another section in the exports.  It is currently segmented for different areas being exported from (e.g. `// Core`).

Reply:
Done!